performance/Bakery-Boulangerie/APABakery.tla

```diff
@@ -270,6 +281,73 @@ Inv == /\ TypeOK
                    \/ pc[nxt[i]] = "e3"
                 => max[nxt[i]] >= num[i]
              /\ (pc[i] = "cs") => \A j \in Procs \ {i} : Before(i, j)
+             
+FIndInv == 
```

Comment:
This doesn't look like just fixing the types. Is this an unrelated change?

Reply:
I guess, I will remove it. It comes from another artifact